recipes/igdiscover/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
     - snakemake >=3.9.0
     - cutadapt
     - xopen >=0.1.1
-    - seaborn >=0.6.0
+    - seaborn =0.7
```

Comment:
Ok, I guessed that that was your reason. Unfortunately, we actually require Seaborn >= 0.6, so now that I’ve looked up the syntax for it, I’ve actually made the requirement even more restrictive.